yarn-project/sequencer-client/src/publisher/l1-publisher.ts

```diff
@@ -272,7 +289,7 @@ export class L1Publisher implements L2BlockReceiver {
       try {
         return await this.txSender.getTransactionReceipt(txHash);
       } catch (err) {
-        this.log(`Error getting tx receipt`, err);
+        //this.log(`Error getting tx receipt`, err);
```

Comment:
Is this for silencing those annoying logs in the e2e tests? If so, can we differentiate between a "receipt not yet available" and an actual error?

Reply:
Hmm, can't remember now. I've uncommented it for now.